DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,21 @@
    "metadata": {},
```

Comment:
<p>Much better! My only remaining suggestions:</p><ul><li>In the "net_sfc_heating" section make it clear that it is approximate *because of* the steady state assumption. Also, I know <code>net_sfc_heating</code> is missing the frazil formation, but I think it gets everything else (at least for the current version of the code, as described in <a href="https://github.com/COSIMA/access-om2/issues/139" target="_blank">https://github.com/COSIMA/access-om2/issues/139</a>)? So I think you should remove the "land-ice heat discharges, and ice calving"? Finally, do you have <code>frazil_3d_int_z</code> for these runs? If so, you could just add it to <code>net_sfc_heating</code> and you've got everything. If not, then note specifically here that this diagnostic is needed.</li><li>I suggest changing "temperature flux at each latitude..." to "a reference-temperature dependent heat flux as there may be net meridional mass transports for which a unique heat transport cannot be defined (see <a href="see [Holmes et al. 2018](https://doi.org/10.1029/2019GL085160" target="_blank">Holmes et al. 2018</a> and <a href="see Holmes et al. 2018 and [Forget and Ferriera 2019](https://www.nature.com/articles/s41561-019-0333-7" target="_blank">Forget and Ferriera 2019</a> for a discussion)."</li><li>I'm not sure what you mean by the "To ensure conservation,..." sentence. Mass is still conserved - there's no issue there. I think you mean something like "To compute a reference-temperature independent heat transport, one would need to subtract the net meridional mass transport first."?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>